src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
+    result: 0,
+
+    add(value) {
+      obj.result += value;
```

Comment:
apply for other methods as well
```suggestion
      this.result += value;
      return this
```